wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -499,11 +504,12 @@ public final class ZMUserSession: NSObject {
         let fileAssetCache = FileAssetCache(location: cacheLocation)
         let userImageCache = UserImageLocalCache(location: cacheLocation)
 
-        self.assetCache = fileAssetCache
+        dependencies.caches.fileAssets = fileAssetCache
+        dependencies.caches.userImages = userImageCache
+        dependencies.caches.searchUsers = NSCache()
```

Comment:
if we're going to move deps to own object, shouldn't it be the responsability of UserSessionDependencies to create those Cache objects ?

Reply:
Good point, was moved to the builder: f3dbf54a962f77959b430462d39c0f607816c1b6